scripts/localk8s/install-app.bat

```diff
@@ -0,0 +1,68 @@
+@ECHO OFF
+SET SCRIPTLOC=%~dp0
+echo Running from: %SCRIPTLOC%
+SET REPOBASE=%SCRIPTLOC%\..\..\..
+echo Other repos should be at: %REPOBASE%
+echo One is:
+dir %REPOBASE%\refarch-kc-order-ms
+
+IF NOT EXIST %REPOBASE%\refarch-kc-order-ms GOTO clone
+GOTO install
+
+:clone
+call %SCRIPTLOC%\..\clone.bat
+
+:install
+:: Create namespace for refarch-kc microservices
+kubectl create ns shipping
+
+:: TODO - is this required outside of OpenShift?
+kubectl create serviceaccount -n shipping kcserviceaccount
+
+:: Configure secrets to allow microservices to connect to Postgres
+kubectl create secret generic postgresql-url --from-literal=binding="jdbc:postgresql://postgresql.postgres.svc:5432/postgres" -n shipping
+kubectl create secret generic postgresql-user --from-literal=binding="postgres" -n shipping
+kubectl create secret generic postgresql-pwd --from-literal=binding="supersecret" -n shipping
+
+:: Create a configmap to allow microservices to find Kafka
+:: This uses port 9092 (without TLS)
+kubectl create configmap kafka-brokers --from-literal=brokers="my-cluster-kafka-bootstrap.kafka.svc:9092" -n shipping
+
```

Comment:
We need to create secret and configmap for Spring Container to be able to connect to BPM: https://github.com/ibm-cloud-architecture/refarch-kc/blob/master/docs/deployments/backing-services.md#bpm